people_manager.rb

```diff
@@ -0,0 +1,79 @@
+require './student'
+require './teacher'
+
+class PeopleManager
+  attr_reader :list_people
+
+  def initialize
+    @list_people = []
+  end
+
+  # People Block
+  def insert_student
+    print 'Insert Name:'
+    name = gets.chomp
+    print 'Insert Age:'
+    age = gets.chomp
+    print 'Has permission? [y, n] '
+    permission = gets.chomp
+    permission = permission == 'y'
+    @list_people.push(Student.new(age: age, name: name, parent_permission: permission))
+    puts 'Student created Succesfully'
+  end
+
+  def insert_teacher
+    print 'Insert Name: '
+    name = gets.chomp
+    print 'Insert Age: '
+    age = gets.chomp
+    print 'Insert Specialization: '
+    spec = gets.chomp
+    parent_auto = true
+    @list_people.push(Teacher.new(age: age, name: name, specialization: spec, parent_permission: parent_auto))
+    puts 'Teacher created Succesfully'
+  end
```

Comment:
- [ ] Let's try to refactor every user input in the manager classes to a single place that will handle only that responsability.